http4k-server/netty/src/main/kotlin/org/http4k/server/Http4kWsChannelHandler.kt

```diff
@@ -58,8 +55,8 @@ class Http4kWsChannelHandler(private val wSocket: WsConsumer) : SimpleChannelInb
 
     override fun channelRead0(ctx: ChannelHandlerContext, msg: WebSocketFrame) {
         when (msg) {
-            is TextWebSocketFrame -> websocket?.triggerMessage(WsMessage(Body(msg.text())))
-            is BinaryWebSocketFrame -> websocket?.triggerMessage(WsMessage(Body(ByteBufInputStream(msg.content()))))
+            is TextWebSocketFrame -> websocket?.triggerMessage(WsMessage(msg.text()))
+            is BinaryWebSocketFrame -> websocket?.triggerMessage(WsMessage(ByteBufInputStream(msg.content()), WsMessage.Mode.Binary))
```

Comment:
I have to wonder whether `CONTINUATION` frames were ever supported.  Maybe this is why `netty` never worked for me.